spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -157,5 +161,21 @@ object HyperConformance extends StreamTransformerFactory {
     }
   }
 
+  private def getReportDate(conf: Configuration): String = {
+    if (conf.containsKey(reportDateKey)) {
+      conf.getString(reportDateKey)
+    } else {
+      DateTime.now().toLocalDate.toString(Constants.ReportDateFormat)
```

Comment:
I believe `org.joda.time` is considered deprecated
```suggestion
import java.text.SimpleDateFormat
import java.util.Date
            new SimpleDateFormat(ReportDateFormat).format(new Date())
```